.repo-metadata-full.json

```
@@ -258,7 +258,7 @@
     "BigQueryDataExchange": {
         "language": "php",
         "distribution_name": "google/cloud-bigquery-data-exchange",
-        "release_level": "preview",
+        "release_level": "stable",
```

BigQueryDataExchange is not stable yet
```suggestion
        "release_level": "preview",
```